webpack/components/AnsibleRoles/components/AnsibleRolesImportButton.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { importUrl } from '../AnsibleRolesPageHelpers';
+import { showToast } from '../../../toastHelper';
+
+export const AnsibleRolesImportButton = () => {
+  let dropdownItems;
+
+  const {
+    response: { results },
+    status,
+  } = useAPI('get', '/api/v2/smart_proxies', {
+    params: { search: 'feature=Ansible' },
+  });
```

Comment:
This is the regular API and returns paginated results. If there are more proxies than fits on a default page, some will be invisible to the user. The previous page used internal APIs that don't paginate, so it would work. It may not be great UX to have a huge list, but this would still be a regression.

Reply:
I added `per_page=all` to the API call.